src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.tsx

```diff
@@ -29,8 +30,19 @@ const CreateFromInstanceType: FC = () => {
           <Card>
             <List className="create-vm-instance-type-section__list" isPlain>
               <SectionListItem
+                headerText={
+                  <>
+                    {t('Select volume to boot from')}{' '}
+                    <HelpTextIcon
+                      bodyContent={t(
+                        'Select a volume (image) to start your VirtualMachine from. You can click the "Add volume" button to add another OS image that doesn\'t appear on the list.',
+                      )}
+                      maxWidth="425px"
```

Comment:
can you please check if it's possible to use the helpIconClassName prop instead of adding this maxWidth prop?

Reply:
Yes, I did and it didn't work as we think it could.

Reply:
So finally I was able to achieve this by using `!important` in the related css file and adding appropriate class to `HelpTextIcon` component. 
Note that suggested using `helpIconClassName` is not appropriate as it is related to the icon itself, not the text that is showed after the icon is clicked.